helm-build-push/action.yaml

```diff
@@ -43,6 +46,8 @@ branding:
 runs:
   using: "composite"
   steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
```

Comment:
This action is designed to work after other actions so we don't really need a checkout here.